src/pybind/mgr/dashboard/HACKING.rst

```diff
@@ -227,7 +227,7 @@ Alternatively, you can use Python's native package installation method::
   $ pip install tox
   $ pip install coverage
 
-To run the tests, run ``tox`` in the dashboard directory (where ``tox.ini``
+To run the tests, run ``run-tox.sh`` in the dashboard directory (where ``tox.ini``
```

Comment:
Please break the line before 80 cols.
```suggestion
To run the tests, run ``run-tox.sh`` in the dashboard directory (where
``tox.ini`` is located).
```

Reply:
@Ranjitha-G can you please amend your PR as requested? Then we're good to go with this one. Thanks!

Reply:
Thanks. Done.

Reply:
Thanks! Could you please squash your commits into one as the final step before we can merge this? Thanks in advance!